patroni/dcs/consul.py

```diff
@@ -122,7 +122,7 @@ class ConsulClient(base.Consul):
     def __init__(self, *args, **kwargs):
         self._cert = kwargs.pop('cert', None)
         self._ca_cert = kwargs.pop('ca_cert', None)
-        self._token = kwargs.get('token')
+        self.token = kwargs.get('token')
```

Comment:
If we rename it here, it also should be renamed everywhere: https://github.com/zalando/patroni/pull/1030/files#diff-01f2f2d2bffe1407b2928d1a79e456b0R135

Reply:
Ah sorry - didn't mean to push yet. Will do 👍